api/taskcluster/webhook.go

```diff
@@ -160,9 +160,14 @@ func handleStatusEvent(ctx context.Context, payload []byte) (bool, error) {
 	// The default timeout is 5s, not enough for the receiver to download the reports.
 	slowCtx, cancel := context.WithTimeout(ctx, resultsReceiverTimeout)
 	defer cancel()
-	var labels []string
+	labels := mapset.NewSet()
 	if status.IsOnMaster() {
-		labels = []string{"master"}
+		labels.Add(shared.MasterLabel)
+	} else {
+		sender := status.GetSender().GetLogin()
+		if sender != "" {
+			labels.Add(sender)
```

Comment:
Similar to a previous discussion of branch names as labels, doesn't this need some prefix since it's effectively user input? `user_`?

Reply:
While I agree with the prospect of potential problem, there's already a precedent for the uploader being one of the labels, so until we see handles like `firefox` or `master` in practice, I don't think we need to worry.

Reply:
Uploaders are manually created. We try our best to avoid ambiguity. For example, we didn't call the Microsoft uploader edge but msedge instead.

User handles, on the other hand, are completely out of our control. firefox, chrome, master, etc. are all existing users. I'm concerned that it would be hard, if not infeasible, to batch modify existing tags later on when we find out there are already name clashes in our Datastore.